src/App.tsx

```diff
@@ -15,48 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Name = 'Sort alphabetically',
+  Length = 'Sort by length',
+  Null = '',
+  Reverse = 'Reverse',
+}
+
+interface Buttons {
+  sortType: SortType,
+  buttonStyle: string,
+}
+
+const buttons: Buttons[] = [
+  { sortType: SortType.Name, buttonStyle: 'is-info' },
+  { sortType: SortType.Length, buttonStyle: 'is-success' },
+  { sortType: SortType.Reverse, buttonStyle: 'is-warning' },
+];
+
+const sortByParams = (
+  toSortArray: string[],
```

Comment:
`toSortArray` means array, that we need to sort, so we can reuse our function without naming "goods" or "products", we can put anything except "goods", so that naming doesn't fit correctly on my opinion